io.c

```diff
@@ -10122,15 +10170,26 @@ rb_io_advise(int argc, VALUE *argv, VALUE io)
 
 /*
  *  call-seq:
- *     IO.select(read_array [, write_array [, error_array [, timeout]]]) -> array or nil
+ *    IO.select(read_ios, write_ios = [], error_ios = [], timeout = nil) -> array or nil
+ *
+ *  Invokes system call {select(2)}[https://linux.die.net/man/2/select],
+ *  which the given I/O objects.
```

Comment:
```suggestion
 *  with the given I/O objects.
```
(if I understand the intention correctly)